src/alfasim_score/common.py

```diff
@@ -52,6 +52,11 @@ class OperationType(str, Enum):
     PRODUCTION = "PRODUCTION"
     INJECTION = "INJECTION"
 
+    @classmethod
+    def _missing_(cls, value):  # type: ignore
```

Comment:
It would be good to have a unit test for it. Thus, improving coverage.

Reply:
Done.